modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -639,6 +639,11 @@
 	<td>xpath=(//label[@for='type']/..//select[@id='type'])[${key_index}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>BUTTON_DELETE_SPECIFIC_ASSIGNMENT</td>
+	<td>//input[contains(@value, '${key_roleName}')]//..//..//..//..//button[contains(@class, 'delete-button')]</td>
```

Comment:
Does `//input[contains(@value, '${key_roleName}')]//button[contains(@class, 'delete-button')]` work instead? We generally don't want to use super specific paths like that because they can be fragile. Basically, if the element structure changes in the future, it's going to break this path very easily.